distribution/lib/Standard/Base/0.0.0-dev/src/Data/Read/Return_As.enso

```diff
@@ -0,0 +1,78 @@
+import project.Any.Any
+import project.Data.Text.Text
+import project.Data.Read.Many_Files_List.Many_Files_List
+import project.Data.Vector.Vector
+import project.Error.Error
+import project.Errors.Common.Type_Error
+import project.Errors.Illegal_Argument.Illegal_Argument
+import project.Function.Function
+import project.Metadata.Display
+import project.Metadata.Widget
+import project.Nothing.Nothing
+import project.Panic.Panic
+from project.Data.Boolean import Boolean, False, True
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Single_Choice
+
+polyglot java import org.enso.base.read.ReadManyReturnSPI
+
+private _get_known_return_classes -> Vector =
+    Vector.from_polyglot_array (ReadManyReturnSPI.get_types False)
+
+## A common interface that represents ways to return a list of files that have
+   been read.
+type Return_As
+    ## PRIVATE
+    Instance underlying
+
+    ## PRIVATE
+    to_text self -> Text = self.underlying.to_text
+
+    ## PRIVATE
+    to_display_text self -> Text = self.underlying.to_display_text
+
+    ## PRIVATE
+    make_return self (input : Many_Files_List) (objects : Vector Any) =
```

Comment:
Why isn't this method `private`?

Reply:
It is used from within `Standard.Table`.